crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -41,6 +41,9 @@ pub struct CallParachainCommand {
 	/// Automatically signs and submits the extrinsic without prompting for confirmation.
 	#[arg(short('y'), long)]
 	skip_confirm: bool,
+	/// Authenticates the sudo key and dispatches a function call with `Root` origin.
+	#[arg(name = "sudo", short = 'S', long)]
```

Comment:
```suggestion
	#[arg(short = 'S', long)]
```
Name _may_ be unnecessary based on the field name being the same.